components/brave_rewards/browser/database/database_contribution_info_publishers.cc

```diff
@@ -157,8 +157,12 @@ bool DatabaseContributionInfoPublishers::InsertOrUpdate(
     return false;
   }
 
-  const std::string query = base::StringPrintf(
-    "INSERT OR REPLACE INTO %s "
+  const std::string query_delete = base::StringPrintf(
```

Comment:
temp fix as we can't do db changes in 1.3. Issue that will resolve it in a right way https://github.com/brave/brave-browser/issues/7838